packages/driver/cypress/integration/e2e/multi-domain/multi_domain_cypress_api.spec.ts

```diff
@@ -240,5 +231,17 @@ describe('multi-domain Cypress API', { experimentalSessionSupport: true }, () =>
         Cypress.Cookies.preserveOnce({})
       })
     })
+
+    it('throws an error when a user attempts to call Cypress.session.clearAllSavedSessions() inside of multi-domain', (done) => {
```

Comment:
Maybe move to the `multi-domain-unsupported-commands` spec?

Reply:
I had the same thought before, but the other `Cypress.*` unsupported methods are in here and this isn't a command...